core/server/master/src/main/java/alluxio/master/file/InodeSyncStream.java

```diff
@@ -977,9 +977,15 @@ private void loadMetadataForPath(LockedInodePath inodePath, RpcContext rpcContex
         && !inodePath.getUri().equals(mRootScheme.getPath())) {
       descendantType = DescendantType.NONE;
     }
+
+    FileSystemMasterCommonPOptions option = NO_TTL_OPTION;
+    if (!Configuration.getBoolean(PropertyKey.MASTER_METADATA_SYNC_IGNORE_TTL)) {
```

Comment:
I don't think we want to update this config at runtime, so it can be moved into a static `boolean` and have better performance.